make-rules/shared-macros.mk

```diff
@@ -170,40 +185,104 @@ PROTOUSRSHAREMAN1MDIR =	$(PROTO_DIR)/$(USRSHAREMAN1MDIR)
 PROTOUSRSHAREMAN3DIR =	$(PROTO_DIR)/$(USRSHAREMAN3DIR)
 PROTOUSRSHAREMAN4DIR =	$(PROTO_DIR)/$(USRSHAREMAN4DIR)
 PROTOUSRSHAREMAN5DIR =	$(PROTO_DIR)/$(USRSHAREMAN5DIR)
+PROTOUSRSHAREMAN8DIR =	$(PROTO_DIR)/$(USRSHAREMAN8DIR)
 PROTOUSRSHARELOCALEDIR =	$(PROTO_DIR)/$(USRSHARELOCALEDIR)
-PROTOUSRKERNELDRVDIR =	$(PROTO_DIR)/$(USRKERNELDRVDIR)
+PROTOUSRKERNELDRVDIR =  	$(PROTO_DIR)/$(USRKERNELDRVDIR)
+PROTOUSRKERNELDRVDIR32 =	$(PROTO_DIR)/$(USRKERNELDRVDIR32)
 PROTOUSRKERNELDRVDIR64 =	$(PROTO_DIR)/$(USRKERNELDRVDIR64)
 
+PROTOUSRBINDIR.32 = 	$(PROTOUSRBINDIR)
+PROTOUSRBINDIR.64 = 	$(PROTOUSRBINDIR64)
+PROTOUSRSBINDIR.32 =	$(PROTOUSRSBINDIR)
+PROTOUSRSBINDIR.64 =	$(PROTOUSRSBINDIR64)
+PROTOUSRLIBDIR.32 = 	$(PROTOUSRLIBDIR)
+PROTOUSRLIBDIR.64 = 	$(PROTOUSRLIBDIR64)
 
 SFWBIN =	/usr/sfw/bin
+SFWBIN32 =	$(SFWBIN)/$(MACH32)
```

Comment:
Not quite true: `pkg search -r '/usr/sfw/bin/amd64/*'` returns quite a few hits, about a dozen are present on my system. Indeed, all are symlinks though. 

When it comes to preferring certain bitness for a build, these vars may be useful; so I'd rather keep this couple (perhaps stripped of mach32).